gulpfile.js

```diff
@@ -60,7 +60,6 @@ gulp.task('createCSS', function() {
     ]))
     .pipe(gulp.dest(opts.destPath))
     .pipe(postcss([
-      autoprefixer(opts.autoprefixer),
```

Comment:
the stream has been already processed with `autoprefixer` at [line#59](https://github.com/daneden/animate.css/pull/566/files#diff-b9e12334e9eafd8341a6107dd98510c9R59)